src/cloud/docker/docker-split-db.md

```diff
@@ -0,0 +1,187 @@
+
+```yaml
+ ...
+mysql-quote:
+    type: mysql:10.2
+    disk: 512
+mysql-sales:
+    type: mysql:10.2
+    disk: 512
+ ...
+```
+
+4. Add the next relationships to the config file `.magento.app.yaml`:
+
+```yaml
+relationships:
+    ...
+    database-quote: "mysql-quote:mysql"
+    database-sales: "mysql-sales:mysql"
+   ...
+```
+
+5. Let's now generate the file by running the cli command:
+```shell script
+php vendor/bin/ece-docker build:compose
+```
+NOTES: As with the 'db' container, you can use the options to set the ports to forward to your local environment:
+  
+```shell script
+php vendor/bin/ece-docker build:compose --expose-db-quote-port=<port for quotee db service> --expose-db-sales-port=<port for sales db service>
+```
+
+6. Аfter generation, open the file `./docker-compose.yml` and you will see that the configuration for database services has been added:
+
+```shell script
+$ cat  ./docker-compose.yml
+...
+services:
+  ...
+  db-quote:
+    hostname: db-quote.magento2.docker
+    image: 'mariadb:10.2'
+    environment:
+      - MYSQL_ROOT_PASSWORD=magento2
+      - MYSQL_DATABASE=magento2
+      - MYSQL_USER=magento2
+      - MYSQL_PASSWORD=magento2
+    ports:
+      - '3306'
+    volumes:
+      - 'magento-db-quote:/var/lib/mysql'
+      - 'docker-entrypoint-quote:/docker-entrypoint-initdb.d'
+      - 'mariadb-conf:/etc/mysql/mariadb.conf.d'
+      - 'docker-mnt:/mnt:delegated'
+    networks:
+      magento:
+        aliases:
+          - db-quote.magento2.docker
+  db-sales:
+    hostname: db-sales.magento2.docker
+    image: 'mariadb:10.2'
+    environment:
+      - MYSQL_ROOT_PASSWORD=magento2
+      - MYSQL_DATABASE=magento2
+      - MYSQL_USER=magento2
+      - MYSQL_PASSWORD=magento2
+    ports:
+      - '3306'
+    volumes:
+      - 'magento-db-sales:/var/lib/mysql'
+      - 'docker-entrypoint-sales:/docker-entrypoint-initdb.d'
+      - 'mariadb-conf:/etc/mysql/mariadb.conf.d'
+      - 'docker-mnt:/mnt:delegated'
+    networks:
+      magento:
+        aliases:
+          - db-sales.magento2.docker
+...
+```
+     
+6. The file `./.docker/config.php.dist` was generated with next configurations:
+
+```php
+<?php
+
+return [
+    'MAGENTO_CLOUD_RELATIONSHIPS' => base64_encode(json_encode([
+       //...
+        'database-sales' => [
+            [
+                'host' => 'db-sales',
+                'path' => 'magento2',
+                'password' => 'magento2',
+                'username' => 'magento2',
+                'port' => '3306'
+            ]
+        ],
+        'database-quote' => [
+            [
+                'host' => 'db-quote',
+                'path' => 'magento2',
+                'password' => 'magento2',
+                'username' => 'magento2',
+                'port' => '3306'
+            ]
+        ],
+    ])),
+   //...
+];
+```
+
+7. Run 
+
+```shell script
+docker-compose up -d
+```
+
+8. Run command 
+
+```shell script
+docker-compose run deploy bash
+```
+
```

Comment:
I don't think that you need the rest of the steps in this topic.

Instead, after step 8 have an ending sentence that says something like: 

After deploying Magento with the split DB configuration, use the database credentials and port information to manage each database. See [Manage the database]({{site.baseurl}}/cloud/docker/docker-manage-database.html.

On the Manage database topic, you could update the sections "Connect to a database" and  "to connect to the database port" to include the split db example.

Reply:
Done